jenkins.yml

```diff
@@ -18,6 +21,8 @@ branches:
       - bedrock-stage
       - bedrock-prod
     integration_tests:
+      - headless
```

Comment:
After a discussion with @alexgibson he showed me that we used to run the firefox and headless tests against production as well. This adds that back.